lib/core/src/lib/search-text/search-text-input.component.scss

```diff
@@ -15,6 +15,10 @@
     }
 
     .adf {
+        &-search-button-inactive {
+            margin-left: -48px;
+        }
+
```

Comment:
this approach using a separate class for the inactive search button state should be more customization-friendly